src/script_chainer/gui/page/script_setting_interface.py

```
@@ -362,7 +368,164 @@ def after_update_item(self) -> None:
+        dialog = PythonCodeEditorDialog(
+            parent=self.window(),
+            title='Python 脚本',
+            initial_code=code,
+            script_path=self.config.script_path,
+        )
+        if dialog.exec() == QDialog.DialogCode.Accepted:
+            self.chain_config.save_python_script(self.config.idx, dialog.get_code())
+            self._update_display()
+
+    def on_delete_clicked(self) -> None:
+        dialog = Dialog('删除脚本', f'确定要删除 {self._get_display_name()} 吗？', parent=self.window())
```

Ruff (0.15.10)

[warning] String contains ambiguous `？` (FULLWIDTH QUESTION MARK). Did you mean `?` (QUESTION MARK)?

(RUF001)